app/views/stages/_fields.html.erb

```diff
@@ -23,23 +23,42 @@
     </div>
   </div>
 
-  <div class="form-group">
-    <div class="col-sm-offset-2 col-sm-10">
-      <div class="checkbox">
-        <%= form.label :confirm do %>
-          <%= form.check_box :confirm %>
-          Confirm before deployment
-        <% end %>
+  <% if ENV['DEPLOY_GROUP_FEATURE'] %>
+    <div class="form-group">
+      <%= form.label :name, 'Deploy Groups', class: 'col-lg-2 control-label' %>
+      <div class="col-lg-4">
+        <%= form.collection_select(:deploy_groups, DeployGroup.all, :id, :name,
+                                   {
```

Comment:
the `# good (normal indent)` example is what I meant:
```
foo(
  bar: 1
)
```

this way there is no wasted space, everything aligns naturally (2,4,6,8 spaces) and changing the initial line does not mean you have to re-indent all following lines